### PR TITLE
unison-manual: Clean up cruft

### LINKED ISSUE #717: Manual has content about obsolete systems (e.g. Windows 98!)
While this is not a big deal, content about obsolete systems is cognitive load and may well be incorrect.
By obsolete systems, I mean those that are so old that no one should be using them:
  * Windows before 7
  * macOS before 10.13
  * UNIX-ish systems that are 1) no longer maintained and 2) for which the most recent formal point release is more than 8 years ago

### PR DESCRIPTION
**Pull request by gdt**
Drop mention of Windows 95.
Drop judgemental terms about TUI vs GUI.
Prune web links that are 404.
Don't use "32-bit" and "64-bit" as if they specify a CPU architecture. Drop content in favor of pointers to BUILDING.md and CONTRIBUTING.md.

Fixes #717

### CODE REVIEW

**Review by tleedjarv — COMMENTED** [review]
I think the removals are good and in the right direction (but can remove even more) but I'm hesitant about some of the additions. In my view, with the current manual we don't need to focus on building from source or even on download instructions. It was a good idea when the HTML manual was available at the homepage. When that is no longer the case then the manual does not have to include anything about downloading and building from source.

File: doc/unison-manual.tex [hunk 1]
```diff
@@ -210,36 +193,23 @@
...
+In principle, Unison should work on any platform to which OCaml has been
+ported and on which the \verb|Unix| module is fully implemented.  It has
+been tested on Windows and a variety of Unix-like systems (FreeBSD,
+GNU/Linux, NetBSD, macOS, Solaris), on multiple CPU types, including
+i386, x86\_64, and earmv7.
 
-If you don't see a pre-built executable for your architecture, you'll
-need to build it yourself.  See \sectionref{building}{Building Unison from Scratch}.
+See {\tt BUILDING.md} in the sources for instructions for building
+Unison for source.
```
[the remark this answers is not in the document]
Comment (gdt) [reply doc/unison-manual.tex]: heavily pruned

File: doc/unison-manual.tex [hunk 2]
```diff
@@ -185,19 +168,19 @@
 
 \SECTION{Installation}{install}{install}
 
-Unison is designed to be easy to install.  The following sequence of
-steps should get you a fully working installation in a few minutes.  If
-you run into trouble, you may find the suggestions on the
-\SHOWURL{http://www.cis.upenn.edu/\home{bcpierce}/unison/faq.html}{Frequently Asked
-Questions page} helpful.  Pre-built binaries are available for a
-variety of platforms.
+Unison is designed to be easy to build from source.
```
[the remark this answers is not in the document]
Comment (gdt) [reply doc/unison-manual.tex]: pruned

File: doc/unison-manual.tex [hunk 3]
```diff
@@ -138,25 +138,8 @@
...
-
-Proposed changes to unison are welcome.
-They should be submitted as pull requests.
-(Since safety and robustness are Unison's most important properties,
-patches will be held to high standards of clear design and clean coding.)
-If you want to contribute to Unison, start by downloading the developer
-tarball from the download page.  For some details on how the code is
-organized, etc., see the file {\tt CONTRIB}.
+See {\tt CONTRIBUTING.md} in the sources for information about
+contributing to Unison.
```
[the remark this answers is not in the document]
Comment (gdt) [reply doc/unison-manual.tex]: redone

File: doc/unison-manual.tex [hunk 4]
```diff
@@ -185,19 +168,19 @@
...
 
 Unison can be used with either of two user interfaces:
 \begin{enumerate}
-\item a simple textual interface, suitable for dumb terminals (and
+\item a textual interface, suitable for terminals (and
 running from scripts), and
-\item a more sophisticated graphical interface, based on Gtk2 (on
-       Linux/Windows) or the native UI framework (on OSX).
+\item a graphical interface, based on GTK3 (on POSIX systems and Windows)
+       and alternatively based on macOS's UI framework.
```
[the remark this answers is not in the document]
Comment (gdt) [reply doc/unison-manual.tex]: pruned

**Comment by gdt** [comment]
I just force-pushed a major rototill addressing your comments, which was mostly removing a lot of content that duplicates what is in sources/wiki, replaced by very very brief pointers.   I am not claiming I fixed everything you objected to, but if not please object again because I did not mean to non-respond to any comments.

File: doc/unison-manual.tex [hunk 5]
```diff
@@ -132,31 +132,37 @@
 \URL{http://research.microsoft.com/\home{fournet}/}{Cedric Fournet}
 contributed to an even earlier prototype.
 
-\TOPSUBSECTION{Mailing Lists and Bug Reporting}{lists}
```
Comment (tleedjarv) [inline doc/unison-manual.tex:135]: Removing this sections breaks a cross-ref in section "Going Further".

File: doc/unison-manual.tex [hunk 6]
```diff
@@ -132,31 +132,37 @@
...
+\end{quote}
+
+The Unison wiki contains information about builds done as part of
+Continuous Integration and other sources of binaries; read the entire
+wiki at:
+\begin{quote}
+\ONEURL{https://github.com/bcpierce00/unison/wiki}
+\end{quote}
+
+\TOPSUBSECTION{Community, Maintenance, and Development}{development}
```
Comment (tleedjarv) [inline doc/unison-manual.tex:150]: See how this section looks when rendered. Using `\paragraph`, like in the old contactsbody.tex, looks slightly better in my opinion.

File: doc/unison-manual.tex [hunk 7]
```diff
@@ -183,69 +189,16 @@
   Synchronization}, and by University of Pennsylvania's Institute for
 Research in Cognitive Science (IRCS).
 
-\SECTION{Installation}{install}{install}
```
Comment (tleedjarv) [inline doc/unison-manual.tex:186]: Removing this sections breaks a cross-ref in section "Local Usage".

File: doc/unison-manual.tex [hunk 8]
```diff
@@ -183,69 +189,16 @@
...
 
+%\finish{Put a pointer somewhere in here to the typical profile in the
+%  reference section.}
 
 \SUBSECTION{Upgrading}{upgrading}
 
-Upgrading to a new version of Unison is as simple as throwing away the old
-binary and installing the new one.
+(This section is perhaps misplaced, but is early because it is far
+better to have at least skimmed it than to not know it exists.)
```
Comment (tleedjarv) [inline doc/unison-manual.tex:201]: Agree, it feels misplaced. Perhaps make it a top-level section?

At some point (but not yet) we can remove all the text about compatibility and interoperability. Then the entire section becomes very short, or can be removed entirely.

File: doc/unison-manual.tex [hunk 9]
```diff
@@ -370,10 +305,10 @@
 adding {\tt -ui text}.
 The other command-line arguments to both versions are identical.
 
-The graphical version can also be run directly by clicking on its icon, but
-this may require a little set-up (see \sectionref{click}{Click-starting
-  Unison}).  For this tutorial, we assume that you're starting it from the
-command line.
+The graphical version can also be run directly by clicking on its
+icon, but this may require a little set-up.
```
Comment (tleedjarv) [inline doc/unison-manual.tex:309]:
```suggestion
The graphical version can also be run directly by clicking on its icon.
```

**Review by tleedjarv — COMMENTED** [review]
I did a fresh review (I didn't go back to my old comments but I think you've covered everything).

There are some reference breaks, but otherwise it's good.

File: doc/unison-manual.tex (hunk 8 above)
Comment (gdt) [reply doc/unison-manual.tex:204]: top it is

File: doc/unison-manual.tex (hunk 9 above)
Comment (gdt) [reply doc/unison-manual.tex]: adopted

File: doc/unison-manual.tex (hunk 6 above)
Comment (gdt) [reply doc/unison-manual.tex:154]: adjusted

File: doc/unison-manual.tex (hunk 5 above)
Comment (gdt) [reply doc/unison-manual.tex:135]: fixed I think

File: doc/unison-manual.tex (hunk 7 above)
Comment (gdt) [reply doc/unison-manual.tex:186]: fixed I think

**Comment by gdt** [comment]
This might actually be ready now.

**Comment by tleedjarv** [comment]
Yes, better merge it. I feel like commit message is no longer accurate but I'll leave it to your judgement.

File: doc/unison-manual.tex [hunk 10]
```diff
@@ -183,69 +197,11 @@
...
-graphical user interface (if you want a GUI at all) on the machine
-where you're actually going to display the interface (the \CLIENT{}
-machine).  Other machines that you synchronize with can get along just
-fine with the textual version.
-
-
-\SUBSECTION{Downloading Unison}{download}
 
-See \ONEURL{https://github.com/bcpierce00/unison/wiki/Downloading-Unison}.
+\SECTION{Upgrading}{upgrading}
```
Comment (tleedjarv) [inline doc/unison-manual.tex:201]: One bug still slipped through. `\SECTION` requires three arguments. The third argument may be left empty. A non-empty third argument seems to make this section a "topic" in the UI (seen by `unison -doc topic` and similarly in the GUI).

**Pull request merged**